vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/total_max_exclusion_rule_unittest.cc

```diff
@@ -67,35 +67,6 @@ TEST_F(BatAdsTotalMaxExclusionRuleTest, AllowAdIfDoesNotExceedCap) {
   EXPECT_FALSE(should_exclude);
 }
 
-TEST_F(BatAdsTotalMaxExclusionRuleTest,
-       AllowAdIfDoesNotExceedCapForMultipleTypes) {
-  // Arrange
-  CreativeAdInfo creative_ad;
-  creative_ad.creative_set_id = kCreativeSetIds.at(0);
-  creative_ad.total_max = 2;
-
-  AdEventList ad_events;
-
-  const AdEventInfo ad_event_1 = GenerateAdEvent(
-      AdType::kAdNotification, creative_ad, ConfirmationType::kServed);
-  ad_events.push_back(ad_event_1);
-
-  const AdEventInfo ad_event_2 = GenerateAdEvent(
-      AdType::kNewTabPageAd, creative_ad, ConfirmationType::kServed);
-  ad_events.push_back(ad_event_2);
-
-  const AdEventInfo ad_event_3 = GenerateAdEvent(
-      AdType::kPromotedContentAd, creative_ad, ConfirmationType::kServed);
-  ad_events.push_back(ad_event_3);
-
-  // Act
-  TotalMaxExclusionRule frequency_cap(ad_events);
-  const bool should_exclude = frequency_cap.ShouldExclude(creative_ad);
-
-  // Assert
-  EXPECT_FALSE(should_exclude);
-}
-
```

Comment:
We can ignore this use case as there will be a change where campaigns will only have a single type